<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="m_pqlabeHiScope">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HiScope:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple panzers killed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_pqlabelSimleAliensKilled">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed panzers killed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_pqlabelSpeedAliensKilled">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Your skope:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_pqlabelYourScope">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_pqlabelTotalKilled">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Game result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_pqlabelGameResult">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_pSpecificationKeys">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shoot: space key&lt;/p&gt;&lt;p&gt;move left: left key&lt;/p&gt;&lt;p&gt;move right: right key&lt;/p&gt;&lt;p&gt;move up: up key&lt;/p&gt;&lt;p&gt;move down: down key&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pqpbNewGame">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
